scripts/model_downloader.py

```diff
@@ -0,0 +1,66 @@
+import requests
+import os
+import wget
+import hashlib
+import time
+
+MODEL_FOLDER = "/models" 
+MODEL_FILENAME = "models.tar.gz"
+MODEL_HOST = os.getenv('COACH_URL', 'coach')
+VERSION_URL = "http://" + MODEL_HOST + "/version"
+FILE_URL = 'http://' + MODEL_HOST + '/' + MODEL_FILENAME
+
+
+
+def try_connect_coach():
+    for _ in range(100):
+        if get_version():
+            print("Coach is available. Continuing config...")
+            return
+        print('Coach is unavailable. Retrying in 1 second')
+        time.sleep(1)
+
+    print('Maximum number of attempts connecting to coach')
+    raise RuntimeError('could not connect to coach')
+
+
+def get_version():
+    try:
+        r = requests.get(url=VERSION_URL)
+        return True
+    except:
+        return False
+
+
+def md5(fname):
+    hash_md5 = hashlib.md5()
+    with open(fname, "rb") as f:
+        for chunk in iter(lambda: f.read(4096), b""):
+            hash_md5.update(chunk)
+    return hash_md5.hexdigest() + "  " + fname + "\n"
+
+def uncompress_models(filename):
+    os.system("tar -xzvf " + filename)
+    os.system("mv src_models " + MODEL_FOLDER)
+
+
+
+try_connect_coach()
```

Comment:
Acho que essa chamada e talvez outras partes do código deveriam estar em um `if __name__ == '__main__'`, tem essa explicação didática [alura](https://www.alura.com.br/artigos/o-que-significa-if-name-main-no-python)

Reply:
Talvez seja interessante colocar dentro de uma classe e chamar os métodos do `ModelDowloader` se isso for relevante pra outras partes. Ou pedir uns conselhos pro @fabiommendes :D

Reply:
Feito! Tinha ficado faltando mesmo